packages/salesforcedx-vscode-lwc/src/testSupport/testRunner/testRunner.ts

```diff
@@ -11,7 +11,12 @@ import * as vscode from 'vscode';
 import { nls } from '../../messages';
 import { telemetryService } from '../../telemetry';
 import { TestExecutionInfo, TestInfoKind } from '../types';
-import { getLwcTestRunnerExecutable, getTestWorkspaceFolder } from './index';
+import {
+  getCliArgsFromJestArgs,
+  getLwcTestRunnerExecutable,
+  workspaceService
+} from '../workspace';
+import { getTestWorkspaceFolder } from './index';
```

Comment:
We should also move `getTestWorkspaceFolder` under the `workspace` module.

Reply:
Good catch - updated.